### DISCUSSION TITLE
Decreasing the target set-point delta? tempnoflap?

### DISCUSSION THREAD

**Discussion by RaW79** [revk/xESP32-Faikout#562] [Q&A]
Hi

I´m exploring the functions and settings of the faikin for the current heating period of the year. My intention is to have smooth ongoing operation of the aircon without rapid big chances in operation. 

Therefore I´m struggeling with the changes the faikin sets to the setpoint of the aircon. These are recognized in my case by a delta of 8 degree (27.5°C to 19.5°C) in one step (brown line).

![grafik](https://github.com/user-attachments/assets/6a9e03fe-f74f-4be2-8b2c-7ab409f8f61d)

Can you recommend how to flatten/decrease this delta? According my understanding the setting heatback is defined on the Minimum temperature of the system. So setting this to e.g. 3 by a Minimum of 19°C will define a heatback-point of 21°C ? Coming e.g. from a setpoint of 27° will also allow a change of 6K. Mmh, also huge in one step. Isn´t this possible to limit by e.g. 3K per minute or sth like this?  

So thought for this intention it is better to increase timenoflap (0 at this time). But the faikin does not accept chances/setting of timenoflap. Are there any operation modes where timenoflap cannot be set in the faikin settings?

Thx in advance!

**Comment by revk** [comment]
Not sure what you mean about not accepting any settings on that. What did you try and what happened?

**Reply by RaW79** [reply]
In the field tempnoflap i change the zero to eg 60 or 120
![grafik](https://github.com/user-attachments/assets/1eff2a53-ecea-466f-965a-d15e7534a146)

pressed Enter to save which did not work 

then changed again to eg 60 or 120 
pressed the save button on the top

again the settings/page was reloaded and in the field of tempnoflap is zero shown like this
![grafik](https://github.com/user-attachments/assets/c710c040-9a34-476c-9e37-2c08e2c341a2)

But nevertheless, can you recommend this for adjusting with the intention to limit the setpoint changes to eg maximum 3K per minute or similar?

**Reply by revk** [reply]
This makes no sense, it works fine for me!

Can you try via MQTT, e.g. `setting/Faikin/tempnoflap 60`

**Reply by RaW79** [reply]
Thx. Yes via MQTT it works. Seems to be an individual problem caused by the browser. Now switched for a test to another browser and it works. Sorry!

Can you recommend a setting to limit the setpoint changes? 

This morning again during heatup (not reached the faikin autocontrol temp bandwith), the setpoint (brown line) was dropped from 27°C to 19.5°C (7.5K) in one shot. (Of course, power (green line) and fan (redline) reacts to this shok).
![grafik](https://github.com/user-attachments/assets/c6fed70d-1406-49c5-9063-43c7c7846496)

Before and afterwards stable operation. Like to avoid giving this big hard shoks to the daikin system.

**Reply by revk** [reply]
It is a time in seconds, so whatever you need.

**Reply by RaW79** [reply]
That´s not really what i need because it will only reduce number of changes within a defined time. It has no impact to change itself.

Therefore I´m asking if there is sth. in the settings?

**Reply by revk** [reply]
This is all covered in the manual I believe, there are a lot of options. You may want the `thermostat` setting, for example.

**Reply by RaW79** [reply]
No I don´t want the thermostat setting. I´m looking for a pid control behavouir which is according manual not available. 
So asking if there are workarounds to reach a limitition of setpoint changes based on value change and not based on fix border limits like e.g. heatback explained in the manual.

**Reply by revk** [reply]
The problem with PID, and why it was not done that way, was because we are fighting the internal operation of the Daikin itself. We are having to second guess how to change the settings to make it work. This was done deliberately by settings several degrees further each way, the settings `coolover`, `coolback`, `heatover`, `heatback` being the main ones. Without these the Daikin sees the new targets as well within the tolerance it has preset for temperature targets and does not actually do anything.

Feel free to play with all such parameters - if you find a better way to get closer to PID, well done, let me know.

**Reply by RaW79** [reply]
Understood Thx! So the big stepwise changes are caused because the Daikin wouldn´t do anything by smaller changes. Means beeing more sensitive will reflect not to more senstive operation by the Daikin -> useless.

**Reply by revk** [reply]
Quite. The alternative is the Daikin's own auto mode, but that does not allow anything like as tight control.

**Reply by RaW79** [reply]
Not clear for me. What do you mean with Daikin´s auto mode? Is there sth. deactivated by the Faikin? For me the Faikin is in it´s simplest explenation an additional control hierachy before the Daikin. As interface or better manipulated variable to the Daikin is only the setpoint of the Daikin used.  Whats happening inside of the Daikin is a black box and always active.

**Reply by revk** [reply]
I mean "Auto" as opposed to "Heat", "Cool", "Fan", etc. Not as good.

We set mode and setpoint. We normally do heat/cool and wide swing on set point. If you don't use the Faikin controls, you can just set mode auto and a setpoint same as using remote control.
